assets/js/app/DataLayer.js

```diff
@@ -724,13 +737,9 @@ LocusZoom.DataLayer.prototype.setElementStatus = function(status, element, toggl
     
     // Set/unset the proper status class on the appropriate DOM element(s)
     d3.select("#" + element_id).classed("lz-data_layer-" + this.layout.type + "-" + status, toggle);
-    if (this.layout.hover_element){
-        var hover_element_class = "lz-data_layer-" + this.layout.type + "-" + this.layout.hover_element + "-" + status;
-        var selector = d3.select("#" + element_id + "_" + this.layout.hover_element);
-        if (this.layout.group_hover_elements_on_field){
-            selector = this.group_hover_elements[element[this.layout.group_hover_elements_on_field]];
-        }
-        selector.classed(hover_element_class, toggle);
+    var element_status_node_id = this.getElementStatusNodeId(element);
+    if (element_status_node_id !== null){
+        d3.select("#" + element_status_node_id).classed("lz-data_layer-" + this.layout.type + "-statusnode-" + status, toggle);
```

Comment:
> Any thoughts on the best way to document this feature for developers?

The natural fit for this would be [in the documentation](https://github.com/statgen/locuszoom/wiki/Data-Layer#adding-custom-methods-and-functionality) where we've got the beginnings of a comprehensive section on authoring custom data layers. Really this section should be expanded to include the basic anatomy of a data layer subclass (including conventions for defining styles in general as well as what style patterns can map to automated behaviors), along with strategies for both developing a new data layer type from scratch as well as expanding an existing one.

I can set about adding to that documentation in general as I'm learning new angles on previously existing challenges in data layer creation with the work on the Manhattan data layer.